free-1.md

```diff
@@ -10,41 +10,92 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Currently, most users of LED strip lights use the default remote to control the colors
+because they don't know how to code anything better. This remote alows for a classic fade
+effect or just picking a color and leaving it there. The only way to upgrade this is by
+using somewhat tricky arduino/raspberry pi code which the average person with LED strip
+lights doesn't know. Therefore, my idea would ideally abstract the complex electrical code
+required and allow for easy customization of very fun and useful LED strip lights.  
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+To create whatever design/controls a user likes, I think a language is necessary
+because there are many features to take advantage of that a simple user interface
+could not handle. For example the user would need a way to integrate time and other
+looping interfaces into the code if they want the lights to follow a certain pattern.
+This would best be solved by a language &mdash specifically a domain specific one as
+they give full responsibility to the users to create complex interactions.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I really like LED strip lights (I have them in my room and am working on putting
+them on my longboard and car) but dont like how the only features are "fades" (going
+from one color to another etc..). I want to grow this technology to allow for more
+complex light creations. For example, I would love for them to know when they should
+turn on and off or to put on a gentle color for sleeping. I came up with this idea at
+4 am talking with a friend about how fun LED lights are. I had just ordered more and my
+friend complained about how they were too repetitive. I realized maybe that could be fixed
+and then double realized that maybe a DSL is the right idea.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Light creations for LED strips
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+A user would interact with the language by linking together commands relevant to their goal.
+I started to imagine an upgraded CS5 sound lab but related to lights and far more fluent.
+I also have wanted to tinker with a language more natural and I think it might be the right
+way to interact with the problem domain since it allows ease of use. On the other hand,
+Loops and aspects of time could be very important to the language so maybe a more technical
+approach is better. I am actually very back and forth on this. 
+
```

Comment:
This is a cool idea and definitely a viable project! A first step might be to think more specifically about the syntax. What experience should the user have when writing programs in this language? What kinds of things do they think about? Are they conscious of time, and do they need to be explicit about it (e.g., do X for five seconds), or are they more interested in coordination of many lights, etc. How might a syntax support users and help them focus on the things they care about (and not worry about the things they don't)?